src/book/01-team/04-history/01-team-history.mdx

```diff
@@ -38,3 +38,66 @@ NUbots made it to the quarter-finals of the competition. Overall we played 13 ga
 | Eliminations for Quarter-Finals (Penalty Shootout) | 3      | 1      | 1   | 0    | 0    |
 | Quarter-Finals                                     | 0      | 1      | 0   | 0    | 1    |
 | Drop-Ins                                           | -5     | 8      | 4   | 1    | 3    |
+
+## 2010: Singapore
+
+**Major hardware changes**
+
+- NAOv3: Intel ATOM 1.6GHz, 1GB Ram,
+
+**Major software changes**
+
+** Performance and Results**
+NUBots were unfortunately knocked out during second round robins, due to a loss with rUNSWift.
+
+| Round              | Points | Played | Won | Drew | Lost |
+| ------------------ | ------ | ------ | --- | ---- | ---- |
+| First Round Robin  | 6      | 2      | 2   | 0    | 0    |
+| Second Round Robin | 2      | 3      | 0   | 2    | 1    |
+
+## 2009: Graz, Austria
+
+**Major hardware changes**
+
+- Aldebaran NAOs get an upgrade (NAOv2), Dual Non-overlapping Cameras, so they can see their feet without having to bend over..
+
+**Major software changes**
+
+- Code Purge, Remove redudent AIBO Code realised by NUPlatform
```

Comment:
```suggestion
- Code purge, removed redundant AIBO Code realised by NUPlatform
```

Reply:
For clarity, do you mean that NUPlatform made some AIBO code redundant, and that code was removed?